sync2jira/upstream_issue.py

```diff
@@ -320,37 +320,47 @@ def github_issues(upstream, config):
 
         issue_updates = config['sync2jira']['map']['github'][upstream].get('issue_updates', {})
 
-        if 'github_project_fields' in issue_updates:
+        if '/pull/' in issue.get('html_url', ''):
```

Comment:
Why do we need to exclude PRs? GitHub allows a PR to be added to a project.

Reply:
Without this exclusion, the error reported at (original) line 336 gets triggered, because the PR is not an issue as far as the `ghquery` is concerned.

Also, there's an entirely separate code flow in [`sync2jira/upstream_pr.py`](https://github.com/release-engineering/Sync2Jira/blob/main/sync2jira/upstream_pr.py) which is ostensibly for handling PRs -- the code here is just for proper Issues.

Reply:
Ok!